README.md

```diff
@@ -1,23 +1,39 @@
-## Important things to know before developing locally or in C9
+## Requirements
+- Python2.7
+- linux, unix, mac, windows(with attachments), Ubuntu
+- virtualenv
+- mysql and python mysql-db
 
-  - You will notice a bunch of sqlite related code, but we are no longer using sqlite
-  - You will need to have mysql and python mysql-db
- 
-# c9 Installation Guide #
+## Setup
+1. In your working environment, clone the URCPP Repo:
+- `https://github.com/BCStudentSoftwareDevTeam/urcpp` as of 02/10/2022
+3. Pull the most recent changes
+4. In the file `setup.sh` add lineRun `source setup.sh`
```

Comment:
I'm not sure what this line means. Also, when I run `source setup.sh` I get an error with the python check:

```
(venv) ramsayb2@Berea-C02ZC049LVDN:~/dev/urcpp-flask %>source setup.sh
Python 2.7.18
You must use Python 2.7. You are using
(venv) ramsayb2@Berea-C02ZC049LVDN:~/dev/urcpp-flask %>
```

Reply:
fixed by removing commenting out line 12-19.